setup.py

```diff
@@ -0,0 +1,25 @@
+from distutils.core import setup
+
+with open("README.md") as readme:
+    README_TEXT = readme.read()
+
+setup(
+    name="tnsmaster",
+    version="0.1.0",
+    description="Toolset for mastering tnsnames",
+    long_description=README_TEXT,
+    author="Dirk Fuchs",
+    author_email="somewhere@over-the-rain.bow",
+    url="https://github.com/difu/tnsmaster",
+    packages=["tnsnames"],
+    data_files=[("tests", ["__init__.py", "test_serviceFinder.py",
```

Comment:
Typically, these would not be included as data files but would be found inside the package they pertain to, see [here](http://the-hitchhikers-guide-to-packaging.readthedocs.org/en/latest/creation.html#arranging-your-file-and-directory-structure)